src/ngAnimate/animateQueue.js

```diff
@@ -491,6 +491,12 @@ var $$AnimateQueueProvider = ['$animateProvider', function($animateProvider) {
       var animateChildren;
 
       while (parent && parent.length) {
+        if (!rootElementDetected) {
+          // angular doesn't want to attempt to animate elements outside of the application
+          // therefore we need to ensure that the rootElement is an ancestor of the current element
+          rootElementDetected = isMatchingElement(parent, $rootElement);
+        }
```

Comment:
The `nodeType` if statement still breaks the loop, but the rootElement is detected beforehand now so that sets `rootElementDetected` to true.